server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -231,40 +242,34 @@ private TrTag renderApplicantRow(
         .with(renderInfoCell(applicant))
         .with(renderApplicantInfoCell(applicant))
         .with(renderActionsCell(applicant))
-        .with(renderDateOfBirthCell(applicant, tiGroup, request));
+        .with(renderDateOfBirthCell(applicant,request));
   }
 
   private TdTag renderDateOfBirthCell(
-      Account account, TrustedIntermediaryGroup tiGroup, Http.Request request) {
+      Account account, Http.Request request) {
 
     Optional<Applicant> newestApplicant = account.newestApplicant();
     if (newestApplicant.isEmpty()) {
-      return td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PR_12);
-    }
-    Optional<LocalDate> current_DOB = newestApplicant.get().getApplicantData().getDateOfBirth();
-    // for cases where DOB is already present, display the Date of Birth
-    // for other cases, show the form to update the DOB.
-    // Note: the update can be done only once.
-    if (current_DOB.isPresent()) {
-      LocalDate unformatedDate = current_DOB.get();
-      return td().with(
-              div(unformatedDate.format(DateTimeFormatter.ofPattern("MM-dd-yyyy")))
-                  .withClasses(Styles.FONT_SEMIBOLD))
-          .withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PR_12);
+      return td().withClasses(BaseStyles.TABLE_CELL_STYLES);
     }
-    return td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.PR_12, Styles.FONT_SEMIBOLD)
+    Optional<LocalDate> currentDob = newestApplicant.get().getApplicantData().getDateOfBirth();
+    return td().withClasses(BaseStyles.TABLE_CELL_STYLES, Styles.FONT_SEMIBOLD)
         .with(
             form()
                 .withClass(Styles.FLEX)
                 .withMethod("POST")
                 .withAction(
-                    routes.TrustedIntermediaryController.updateDateOfBirth(tiGroup.id, account.id)
+                    routes.TrustedIntermediaryController.updateDateOfBirth(account.id)
                         .url())
                 .with(
-                    input().withType("date").withName("dob").withPlaceholder("Click to Enter"),
+                    input()
+                        .withId("date-of-birth-update")
+                        .withName("dob")
+                        .withType("date")
+                        .withValue(currentDob.isPresent() ? currentDob.get().toString() : ""),
                     makeCsrfTokenInputTag(request),
-                    submitButton("Update")
-                        .withClasses(Styles.P_0, ApplicantStyles.BUTTON_PROGRAM_APPLY)));
+                    submitButton("add DOB")
```

Comment:
Even though we uppercase the text visually, I think we should capitalize the "Add"